accord-core/src/main/java/accord/utils/RandomSource.java

```diff
@@ -350,7 +350,7 @@ default long reset()
 
     default Random asJdkRandom()
     {
-        return new Random()
+        return new Random(nextLong())
```

Comment:
this was a bug that impacted deterministic execution.  When we fork to a java `Random` we didn't control the seed so could have different histories cross runs with the same seed!